src/lib/providers/parser.ts

```
@@ -999,3 +999,206 @@ function transformTraktRecommendations(
+}
+
+export interface CollectionDetails {
+    id: number;
+    name: string;
+    overview: string | null;
+    poster_path: string | null;
+    backdrop_path: string | null;
+    parts: CollectionMovie[];
+}
+
+export function parseCollectionDetails(collectionData: any): CollectionDetails {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)